ext/js/background/backend.js

```diff
@@ -1034,10 +1036,10 @@ export class Backend {
     }
 
     /**
-     * @param {undefined|{mode: 'existingOrNewTab'|'newTab'}} params
+     * @param {undefined|{mode: 'existingOrNewTab'|'newTab'|'popup'}} params
      */
     async _onCommandOpenSettingsPage(params) {
-        /** @type {'existingOrNewTab'|'newTab'} */
+        /** @type {'existingOrNewTab'|'newTab'|'popup'} */
         let mode = 'existingOrNewTab';
```

Comment:
Do you want to make `mode` an enum in this PR or separate?

Reply:
Can make another pr for that later.